source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,324 @@
+
+Direct Debit payments are an asynchronous payment method. When a user
+completes a Direct Debit payment process using GOV.UK Pay, the platform makes
+a request to their bank to set up the Direct Debit mandate and transfer the
+funds. It takes 3-4 working days to confirm the success of the payment.
+
+## Get started
+
+To use the GOV.UK Pay Direct Debit feature, you must request a GOV.UK Pay
+Direct Debit test account [from our support
+team](/support_contact_and_more_information/#contact-us).
+
+If you want to add Direct Debit to an existing integration, you should
+[contact us](/support_contact_and_more_information/#contact-us) to discuss
+this.
+
+### Set up an account with the Direct Debit supplier
+
+Select your account in the __My services__ page of your GOV.UK Pay Direct
+Debit account then follow the on-screen instructions.
+
+### Control the payment method
+
+When you create an API key, you link your GOV.UK Pay account with the payment
+method you want to use. For example “testing”, “Worldpay”, or
+“Direct Debit”.
+
+You can use different API keys to direct paying users to
+different payment methods.
+
+## GOV.UK Pay ‘agreements’
+
+In order to support Direct Debit, GOV.UK Pay uses ‘agreements’. An
+agreement represents an individual payer’s consent for future payments using a
+particular funding source. For example, a personal bank account.
+
+For Direct Debit transactions, GOV.UK Pay fulfils an agreement by registering
+a Direct Debit mandate with the payer’s bank.
+
+### Agreement lifecycle
+
+1. `CREATED` - Your service made an API call to create the agreement
+
+2. `STARTED` - Payer has visited the `next_url` and landed on the GOV.UK Pay
+page to enter their account details
+
+3. `PENDING` - Indicates that account details have been submitted to the
+payment service provider (PSP), but the PSP is not yet able to confirm whether
+the mandate has been successfully set up
+
+4. The agreement then reaches 1 of 5 different finished states
+
+#### Possible agreement finished states
+
+<br>
+<div style = "overflow-x:auto;">
+  <table style = "width:100%">
+    <tr>
+    <th nowrap>Finished state</th>
+    <th>Description</th>
+    </tr>
+    <tr>
+    <td nowrap><code>SUCCESS</code></td>
+    <td nowrap>Mandate successfully set up.</td>
+    </tr>
+    <td nowrap><code>FAILED</code></td>
+    <td nowrap>User attempted to make a payment but the mandate was not set up successfully.</td>
+    </tr>
+    <td nowrap><code>CANCELLED</code></td>
+    <td nowrap>The user has cancelled their mandate.</td>
+    </tr>
+    <td nowrap><code>CANCELLED - USER NOT ELIGIBLE</code></td>
+    <td nowrap>The user indicated that they are not the sole signatory on the bank account, so the mandate setup could not be completed.</td>
+    </tr>
+    <td nowrap><code>ERROR</code></td>
+    <td nowrap>Something went wrong with GOV.UK Pay or the underlying Payment Service Provider.</td>
+    </tr>
+  </table>
+</div>
```

Comment:
could this table be wrapped? it's not immediately obvious that you need to scroll to see all the text